Resources/ModelingAndBinding-Starter/Podfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment the next line to define a global platform for your project
+platform :ios, '10.0'
+
+use_frameworks!
+
+pod 'IGListKit', :git => 'https://github.com/Instagram/IGListKit.git', :branch => 'master'
+pod 'SDWebImage'
```

Comment:
Ya, we can. I don't want to promote bad practices, but its probably fine to just hack it and download here